.github/workflows/test-and-deploy.yml

```diff
@@ -43,6 +45,19 @@ jobs:
       - name: Run Unit Tests
         run: make test
 
+      - name: Fix code coverage paths
+        if: ${{ (github.event_name == 'pull_request' || github.ref_type == 'branch') && matrix.ruby == '3.0' && !github.event.pull_request.head.repo.fork }}
+        working-directory: ./coverage
+        run: |
+          sed -i 's@'$GITHUB_WORKSPACE'@/github/workspace/@g' coverage.json
+
+      - name: SonarCloud Scan
+        if: ${{ (github.event_name == 'pull_request' || github.ref_type == 'branch') && matrix.ruby == '3.0' && !github.event.pull_request.head.repo.fork }}
```

Comment:
This should be running SonarCloud on Ruby 3.0, but it looks like it's being skipped on that version too: https://github.com/twilio/twilio-ruby/runs/4859609125?check_suite_focus=true

Reply:
It skipped it because the source branch is from my fork otherwise it would blow up because of invalid tokens. You can see a valid run on my fork: https://github.com/BrimmingDev/twilio-ruby/actions/runs/1714902065.